basics/generate_pdf.php

```diff
@@ -0,0 +1,26 @@
+<?php
+function generate_pdf($fullName,$email_address,$number,$image,$marks_data): void
```

Comment:
1/ Multiple params should be separate by a comma and a space.
```
generate_pdf($fullName, $email_address, $number, $image, $marks_data)
```
2/ Follow a uniform structure for variables.
Normally class variables are camel cased, eg. $userName.
And function variables are snake cased, eg. $user_name.
So as in this file, there are not classes, all variables will be snake cased.
Please make similar changes in other files also.